src/library/objects/Node.js

```diff
@@ -662,6 +662,13 @@ Used by SortieManager
 			this.unexpectedList.push(...this.unexpectedDamagePrediction(result.fleets.playerEscort,
 				1, battleData.api_formation[0], battleData.api_formation[2], isRealBattle)
 			);
+
+			// Check for any one-time special cutin
+			const checkSortieSpecialAttack = attacks => attacks.some(attack => (attack.cutin || attack.ncutin) >= 100);
+			const playerMain = result.fleets.playerMain;
+			if (checkSortieSpecialAttack(playerMain[0].attacks)) {
+				this.sortieSpecialAttack = playerMain.map(ship => checkSortieSpecialAttack(ship.attacks));

```

Comment:
btw, this one cannot check out the combined attackers since only flagship get 100/101

Reply:
BP module should already assign 100/101 to attackers involved

Reply:
yea, my fault. my tests always leave 1 enemy to be killed by nagato 😂